core/src/main/scala/coulomb/rational/rational.scala

```diff
@@ -78,6 +78,8 @@ final class Rational private (val n: BigInt, val d: BigInt) extends Serializable
         case v: Long => (n == v) && (d == 1)
         case _ => false
 
+    override def hashCode: Int = 29 * (37 * n.## + d.##)
```

Comment:
Another bug, caught by laws. I copied this from spire.

https://github.com/typelevel/spire/blob/b8d5c71b499e7e477a0abed1db344298b31c3be5/core/src/main/scala/spire/math/Rational.scala#L737-L739

Reply:
:+1: